phpstan-baseline.neon

```diff
@@ -13252,74 +13252,15 @@ parameters:
 			path: app/cdash/include/api_common.php
 
 		-
-			message: "#^Argument of an invalid type array\\|false supplied for foreach, only iterables are supported\\.$#"
-			count: 2
-			path: app/cdash/include/autoremove.php
-
-		-
-			message: """
-				#^Call to deprecated function add_last_sql_error\\(\\)\\:
-				04/22/2023$#
-			"""
+			message: "#^Dynamic call to static method Illuminate\\\\Database\\\\Eloquent\\\\Builder\\<App\\\\Models\\\\Build\\>\\:\\:limit\\(\\)\\.$#"
```

Comment:
Any thoughts on what to do about this (if anything?)

Reply:
That's unfortunately a side effect of Laravel's backend magic which allows you to call methods both statically and dynamically.  For now, I've just been ignoring them.  Once we have a significant number of ignored errors like this, I'll look into configuring PHPStan to ignore these for Eloquent-derived classes.